fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/FedoraTypesUtils.java

```diff
@@ -153,6 +168,12 @@
     });
 
     /**
+     * Check if a property is intentionally suppressed.
+     */
+    final static List<String> suppressed = Arrays.asList("jcr:mimeType", "jcr:lastModified");
```

Comment:
May as well use the constant defined for `jcr:lastModified`: FedoraJcrConstants.JCR_LASTMODIFIED.

You are welcome to add a new FedoraJcrConstants.JCR_MIMETYPE if you like a consistent list of elements.